### PR TITLE
Add mkosi project to website

### PR DESCRIPTION
**Pull request by hulkoba**
# Add mkosi project to systemd website 
Put it to 'Project' section

_Note_ that https://github.com/systemd/mkosi/pull/2324 needs to be merged first, otherwise the website won't exist :)

### CODE REVIEW

File: docs/_data/extra_pages.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [
+    { "category": "Project", "title": "Mkosi project - Build Bespoke OS Images", "url": "https://mkosi.systemd.io/" },
```
Comment (poettering) [inline docs/_data/extra_pages.json]: project should probably be uppercase.

Comment (poettering) [reply docs/_data/extra_pages.json]: and `mkosi` is probably better spelled in lowercase, as per UNIX tradition

**Comment by poettering** [comment]
besides that, lgtm

**Comment by poettering** [comment]
given that https://mkosi.systemd.io/ is now up i guess this could be merged too. @hulkoba can you make the requested changes, should be able to land quickly then

**Comment by hulkoba** [comment]
@poettering Sorry for the late response. I thought we merge #31178 first and then rebase this branch because there are changes in the structure of the docs.
I adapted the text and this can be merged now :rocket:

**Review by DaanDeMeyer — APPROVED** [review]

**Pull request merged**